images/router/haproxy/conf/haproxy-config.template

```diff
@@ -226,6 +226,11 @@ backend be_edge_http_{{$cfgIdx}}
     {{ else }}
   balance leastconn
     {{ end }}
+    {{ with $value := index $cfg.Annotations "router.openshift.io/haproxy.timeout"}}
+      {{if (matchString "[0-9]*(us|ms|s|m|h|d)?" $value) }}
```

Comment:
'rm -f *; 90ms test' will match. the regex needs to be ^[1-9][0-9]*(us|ms|s|m|h|d)?$